src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -47,6 +51,9 @@ public static class ScriptHostBuilderExtensions
         private const string DelayedConfigurationActionKey = "MS_DelayedConfigurationAction";
         private const string ConfigurationSnapshotKey = "MS_ConfigurationSnapshot";
 
+        private const string ExternalStartupErrorDiagnosticId = "AZFD0005";
```

Comment:
I think we only have one other place in host now (HostIdValidator) that is emitting diagnostic logs? I wonder if we shouldn't move the diagnostic IDs and help links into a central file so they're all in one place.